Cargo.toml

```diff
@@ -29,13 +29,13 @@ shred = { git = "https://github.com/torkleyy/shred" }
 shred-derive = "0.1.2"
 tuple_utils = "0.2"
 
-serde = { version = "0.9.11", optional = true }
-serde_derive = { version = "0.9.11", optional = true }
+serde = { version = "1.0.8", optional = true }
```

Comment:
what's the reason for specifying "1.0.8" as opposed to just "1.0" here and for `serde_derive`?